app/bundles/CoreBundle/DataFixtures/ORM/NotificationFixtures.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Mautic\CoreBundle\DataFixtures\ORM;
+
+use DateTime;
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\DataFixtures\OrderedFixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use Mautic\CoreBundle\Entity\Notification;
+
+class NotificationFixtures extends AbstractFixture implements OrderedFixtureInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getOrder()
+    {
+        return 3;
+    }
+
+    /*
+     * {@inheritdoc}
+     */
+    public function load(ObjectManager $manager)
+    {
+        $user = $this->getReference('admin-user');
+        for ($i = 0; $i < 500; ++$i) {
+            $n = new Notification();
+            $n->setHeader('notification'.$i);
+            $n->setMessage('lorem ipsum');
+            $n->setDateAdded($this->randomDateInRange(new \DateTime('-8 day'), new DateTime()));
+            $n->setUser($user);
+            $manager->persist($n);
```

Comment:
I know that this is fixtures and no biggie. But batching your flushes is great for performance.

Reply:
Noted.